recipes/bgen/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "bgen" %}
+{% set version = "0.1.14" %}
+{% set sha256 = "311d1733ef15de633ddcb52605a0f5b6999c8c2e07afc8c67348ae026431c36e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/limix/bgen/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win32 or linux32 or (win64 and py27)]
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py36]
+
+requirements:
+  build:
+    - python  # [win]
+    - toolchain
+    - cmake
+    - zstd
+    - zlib  # [win]
+  run:
+    - zstd
+    - zlib  # [win]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
```

Comment:
You need to add

```yaml
    - vc 9  # [win and py27]
    - vc 10  # [win and py34]
    - vc 14  # [win and (py35 or py36)]
```

to both build and run time dependencies so the right `vc` is installed when you are using the package.
With that you can, and should, remove the hack:

```yaml
 requires:
    - python {{ environ['PY_VER'] + '*' }}  # [win]
```